### PR TITLE
Markdown style improvements

### PR DESCRIPTION
**Pull request by jancborchardt**
* remove ugly underline from Markdown headings
* improve look of Markdown link underline, don’t cut through text
* improve look and clickability of Markdown checkbox

Please review @Raydiation @LukasReschke :)

### CODE REVIEW

**Comment by jancborchardt** [comment]
Review please @Raydiation @LukasReschke @DavidPrevot @fcturner @vgezer :)

File: css/notes.css [hunk 1]
```diff
@@ -138,20 +138,27 @@
...
 
 #app-content .utils {
-    padding: 0 20px 0 20px;
-    line-height: 30px;
     position: fixed;
-    top: 45px;
-    right: 20px;
+    top: 58px;
+    right: 0;
+    -ms-filter: "progid:DXImageTransform.Microsoft.Alpha(Opacity=50)";
```
Comment (Raydiation) [inline css/notes.css]: ms filters are not needed ;)

Comment (jancborchardt) [inline css/notes.css]: Why not – does the app not work in IE?

Comment (Raydiation) [inline css/notes.css]: Your filter only works in 8 or lower which is not supported (and it will fail)

Comment (jancborchardt) [inline css/notes.css]: Amended and force-pushed :)

File: css/notes.css [hunk 2]
```diff
@@ -138,20 +138,27 @@
 }
 
 #app-content .utils {
-    padding: 0 20px 0 20px;
-    line-height: 30px;
     position: fixed;
-    top: 45px;
-    right: 20px;
+    top: 58px;
+    right: 0;
```
Comment (Raydiation) [inline css/notes.css]: This puts the checkbox to the very right which makes the text hard to read (especially when a scroll bar is present)

Comment (jancborchardt) [inline css/notes.css]: No, as some lines below there is padding added to the label. Did you actually try this out? ;)

Comment (Raydiation) [inline css/notes.css]: Yep tried it out :)

Comment (Raydiation) [inline css/notes.css]: padding right is not applied for me I think

Comment (Raydiation) [inline css/notes.css]: Ok, looks fine in Chrome but padding issue in Firefox

Comment (jancborchardt) [inline css/notes.css]: FML

Comment (jancborchardt) [inline css/notes.css]: Looks fine for me in both Chrome and Firefox:
capture du 2015-07-09 14-21-06

@Henni?

**Comment by jancborchardt** [comment]
@Raydiation fixed it. Review again please.

**Comment by jancborchardt** [comment]
@Raydiation @Henni I fixed it. Review again please.

**Comment by Raydiation** [comment]
:+1: good job :)

**Pull request merged**